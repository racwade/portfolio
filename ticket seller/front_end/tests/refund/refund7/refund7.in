login
test_admin
refund
test_buy
test_sell
990001
logout