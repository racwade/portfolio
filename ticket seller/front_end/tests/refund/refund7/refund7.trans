05 test_buy      test_sell      000990001
00 test_admin      AA 000000999