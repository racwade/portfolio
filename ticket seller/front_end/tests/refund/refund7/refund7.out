welcome to the ticket seller system
enter username:
logged in as test_admin
enter event name:
enter buyer name:
enter seller name:
enter refund amount:
thank you $990001 transferred to test_buy from test_sell
bye